Project2015To2017/ProjectConverter.cs

```diff
@@ -15,6 +15,14 @@ public class ProjectConverter
 	{
 		private static IReadOnlyList<ITransformation> TransformationsToApply(ConversionOptions conversionOptions)
 		{
+			if (conversionOptions.AnalyzeOnly)
+			{
+				return new ITransformation[]
+				{
+					new FileTransformation(),
```

Comment:
That's horrible, I believe we must rewrite `IncludeItems` mechanics completely, but for now that's a necessity.